plugins/modules/azure_rm_containerregistry_tag.py

```diff
@@ -0,0 +1,349 @@
+        type: str
+    registry:
+        description:
+            - The name of the container registry.
+        type: str
+        required: true
+    repository_name:
+        description:
+            - The name of the repository within the registry.
+            - Required when state = C(absent).
+            - If omitted when state = C(present), the name of the source repository will be used.
+        type: str
+    name:
+        description:
+            - The name of the tag.
+            - If omitted when state = C(present), the name of the source tag will be used.
+            - If omitted when state = C(absent), the whole repository will be deleted.
+        type: str
+    source:
+        description:
+            - The source image detail. Required when state = C(present).
+        type: dict
+        suboptions:
+            registry_uri:
+                description:
+                    - The address of the source registry.
+                type: str
+            repository:
+                description:
+                    - Repository name of the source image.
+                type: str
+                required: true
+            name:
+                description:
+                    - Name of the tag.
+                type: str
+                default: latest
+            credentials:
+                description:
+                    - Credentials for the source registry.
+                type: dict
+                suboptions:
+                    username:
+                        description:
+                            - Username for the source registry
+                        type: str
+                    password:
+                        description:
+                            - Password for the source registry
+                        type: str
+    state:
+        description:
+            - Assert the state of the tag.
+        type: str
+        default: present
+        choices:
+            - present
+            - absent
+
+extends_documentation_fragment:
+    - azure.azcollection.azure
+    - azure.azcollection.azure_tags
+
+author:
+    - Ross Bender (@l3ender)
+'''
+
+EXAMPLES = '''
+- name: Import a tag
+  azure_rm_containerregistry_tag:
+    registry: myRegistry
+    source_image:
+      registry_uri: docker.io
+      repository: library/hello-world
+      name: latest
+
+- name: Import a tag to a different name
+  azure_rm_containerregistry_tag:
+    registry: myRegistry
+    repository_name: app1
+    name: v1
+    source_image:
+      registry_uri: docker.io
+      repository: library/hello-world
+      name: latest
+
+- name: Delete all tags in repository
+  azure_rm_containerregistry_tag:
+    registry: myRegistry
+    repository_name: myRepository
+    state: absent
+
+- name: Delete specific tag in repository
+  azure_rm_containerregistry_tag:
+    registry: myRegistry
+    repository_name: myRepository
+    name: myTag
+    state: absent
+'''
+
+RETURN = '''
```

Comment:
Missing return value
```suggestion
RETURN = '''
```

Reply:
The module does not return any values (only `changed`). Can you advise on what to put in the documentation for return?